extensions/amp-next-page/1.0/test/test-amp-next-page.js

```diff
@@ -617,6 +617,56 @@ describes.realWin(
           ).to.equal('cid');
         });
       });
+
+      it('shadow-doc Viewer calls host-doc messageDeliverer', async () => {
+        const initialMessageDeliverer = env.sandbox.spy();
+        const initialViewer = Services.viewerForDoc(ampdoc);
+        initialViewer.setMessageDeliverer(initialMessageDeliverer, '');
+
+        await fetchDocuments(service, MOCK_NEXT_PAGE, 2);
+
+        for (const index of [1, 2]) {
+          const {ampdoc} = service.pages_[index].shadowDoc;
+          const viewer = Services.viewerForDoc(ampdoc);
+          expect(viewer).to.not.equal(initialViewer);
+
+          const messageDeliverer = viewer.maybeGetMessageDeliverer();
+          expect(messageDeliverer).to.be.ok;
```

Comment:
Per chai docs  it is not recommended to use `ok` (https://www.chaijs.com/api/bdd/#method_ok). Use `not.to.be.null` in this case?